WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/HttpRest/ResponseHelpers.cs

```diff
@@ -84,11 +84,11 @@ public static T ReadJsonStream<T>(Stream dataStream, DataContractJsonSerializerS
                     {
                         response = serializer.ReadObject(dataStream);
                     }
-                    catch (SerializationException)
+                    catch (SerializationException e)
```

Comment:
Should this be Exception instead of SerializationException?  That way, we can catch/report anything that goes wrong.

Reply:
Makes sense.